include/spdlog/sinks/ansicolor_sink.h

```diff
@@ -0,0 +1,116 @@
+//
+// Copyright(c) 2016 Kevin M. Godby (a modified version by spdlog).
```

Comment:
Seems like we would need to call this out similar to https://github.com/apache/nifi-minifi-cpp/blob/ff7aa7fc3bf55681398427f1de6af7f352a100fa/LICENSE#L394